### PR TITLE
[#10941] Copying questions requires scrolling the whole list of current questions 

### LINKED ISSUE #10941: Copying questions requires scrolling the whole list of current questions
Screenshot from 2021-01-30 15-26-02

Copying questions requires scrolling the whole list of current questions which could be time-consuming if the list is long. The checkbox for each question appears to be very small on my laptop.

Suggestions:
- Fix the buttons' positions e.g bottom of the modal
- The click event for checkbox can be created by clicking the row

### PR DESCRIPTION
**Pull request by Jellwood**
Fixes #10941 

**PR Checklist**

**Outline of Solution**

The footer was attached by moving all of the styling of the modal footer and the table in the modal window within the super CSS file for the modal window for copying. Once the footer was the appropriate width, the height was fixed to allow for a table border the same height so that the footer does not interfere with the last question. The behavior was preserved from the clip shown in the issue and the table did not change any functionalities.

### CODE REVIEW

**Comment by madanalogy** [comment]
Hello thanks for contributing to teammates! Could you please record a short gif/vide demonstrating your fix for the modal?

**Comment by Jellwood** [comment]
Hi! I am so sorry about that, this GIF shows the process of copying a question in addition to scrolling through the table until the end to show the table formatting, and finally selecting to ensure that the functionality still works!

TeammatesWorkingStickyFooter

**Review by t-cheepeng — APPROVED** [review]
LGTM

**Review by Derek-Hardy — CHANGES_REQUESTED** [review]
The code looks fine to me. 

However, the original issue has another small checkbox fix. You can either attempt it here or modify PR description to 'Part of #10941'.

**Comment by Jellwood** [comment]
Sorry about that, I'll try to get it working and having the checkboxes size up

**Comment by Jellwood** [comment]
I was able to get the checkbox elements to a larger size for Chrome, I tested using Chrome and Firefox, and the boxes are similar sizes in both browsers. I was able to replicate the smaller checkbox sizes in Chrome, and I used that as the baseline. I do not have access to Safari to ensure that it works in this browser, and I can build in my Windows environment this weekend as well to make sure that Internet Explorer also works with these new additions if needed. The GIF below shows my Firefox browser then my Chrome browser to show that both browsers are displaying similar sized checkboxes.
TeammatesBoxesFirefoxChrome

File: src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+  height: 50px;
+  width: 69.8%;
+  justify-content: right;
+  background-color: white;
+}
+
+td {
+  text-align: center;
+  vertical-align: middle;
+}
```
Comment (Derek-Hardy) [inline src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss:22]: I am not entirely sure the central alignment improves the readability. I will leave this to other reviewers' opinions.

**Original:**
Screenshot 2021-02-19 at 13 34 55

**After:**
Screenshot 2021-02-19 at 13 35 41

**Review by Derek-Hardy — APPROVED** [review]
Checkbox scaling looks good to me.

File: src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss (hunk 1 above)
Comment (moziliar) [reply src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss:22]: Speaking for myself: I'd prefer the original look. That allows reading in a more natural manner.

Comment (madanalogy) [reply src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss:22]: For UI decisions such as this one, best to check with @damithc

Comment (t-cheepeng) [reply src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss:22]: Previously we have standardized that such tables with text should be left-aligned (i.e. original)

Comment (Jellwood) [reply src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss:22]: I can make those changes and have them in tomorrow if need be. That was my bad, I forgot that the earlier demo questions tended to have shorter questions that appeared to have centered check boxes and I did not want to break that alignment in adding scaling. If the original left-aligned is the standard, I can go back and make sure everything fits that standard.

Comment (damithc) [reply src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss:22]: @t-cheepeng was involved in standardizing this kind of UI issues. @t-cheepeng do we have a document of the decisions taken? If not, we should.

Comment (t-cheepeng) [reply src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss:22]: @damithc Yes, we do have a document for this. But it was not made made public/posted on github

Comment (damithc) [reply src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/copy-questions-from-other-sessions-modal.component.scss:22]: > @damithc Yes, we do have a document for this. But it was not made made public/posted on github

I see. Let's add it to our docs. At least a public link to the Google doc.

**Review by t-cheepeng — APPROVED** [review]

**Review by madanalogy — APPROVED** [review]

**Comment by madanalogy** [comment]
Thank you for your contribution! :)

**Pull request merged**